test/jdk/java/lang/reflect/code/TestStringTemplateOp.java

```diff
@@ -0,0 +1,56 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.code.Op;
+import java.lang.reflect.code.interpreter.Interpreter;
+import java.lang.reflect.code.op.CoreOps;
+import java.lang.runtime.CodeReflection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+/*
+ * @test
+ * @enablePreview
+ * @run testng TestStringTemplateOp
+ */
+
+public class TestStringTemplateOp {
+
+    @CodeReflection
+    static String f(int x, int y) {
```

Comment:
Add a test that includes a conditional expression in a fragment expression.

Would be also useful to add a tests
- where fragment expressions return values of all primitive types and a reference type; and
- where the number of expressions is greater than the `List.of` parameter threshold where varargs is used.